cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -1076,47 +1033,20 @@ def _validate_raw(self):
         # If all checks passed then proceed with validation
         if all(checks):
 
-            normalization = self.adata.uns["X_normalization"]
-
             # If both "raw.X" and "X" exist but neither are raw
             if not self._is_raw() and self._get_raw_x_loc() == "raw.X":
                 self.warnings.append(
```

Comment:
@brianraymor should this be an error?
```suggestion
                self.error.append(
```